cmd/encryption-v1.go

```diff
@@ -55,7 +55,8 @@ var (
 	// Additional MinIO errors for SSE-C requests.
 	errObjectTampered = errors.New("The requested object was modified and may be compromised")
 	// error returned when invalid encryption parameters are specified
-	errInvalidEncryptionParameters = errors.New("The encryption parameters are not applicable to this object")
+	errInvalidEncryptionParameters     = errors.New("The encryption parameters are not applicable to this object")
+	errInvalidEncryptionParametersSSEC = errors.New("SSE-C encryption parameters are not supported on this bucket")
```

Comment:
Should we say not supported on a replicated bucket

Reply:
That is returned to the client @kannappanr this is just a placeholder error.

Reply:
> mc: <ERROR> Failed to copy `/etc/hosts`. SSE-C encryption parameters are not supported on replicated bucket.